### DISCUSSION TITLE
IDE64?

### DISCUSSION THREAD

**Discussion by jokey-carrot** [frntc/Sidekick64#53] [Ideas]
Any chance of IDE64 functionality coming to this project? Lots of games and software have been converted and it's even compatible with geos. 
Happy Christmas - From George

**Comment by frntc** [comment]
Hi George,

I've never had a chance to hold a IDE64 in my hands, and even didn't try its Vice emulation (but it seems to be an interesting piece of hw). Judging from the Vice source code, it's quite some work to emulate it (and I could not find a detailed documentation of the hardware -- let me know if you have a link!). I'm curious to read more about it, but given the already-on-the-list ideas for my projects and little free time, I can't make promises that any functionality can happen anytime soon :-/

**Reply by mrjoso** [reply]
Hi Carsten, I'm Josef from IDE64 Project.
I can provide any information about IDE64 cartridge if you decide to emulate it in Sidekick64.
IDE64 project is maintained, but the new IDE64 cartridges are not made because of the remaining integrated circuit shortage.

**Comment by Kayto** [comment]
Hi all and a HNY!
I have been on the look out for an IDE64 for a while. It does not look like it is actively developed anymore - shame. It did look interesting.
[http://singularcrew.hu/idedos/IDE64_users_guide.pdf](url)

**Reply by jokey-carrot** [reply]
The Ide64 isn't sold anymore because the people making it can't get ahold of the required ICs.
